test/fixtures/users.yml

```diff
@@ -2,11 +2,11 @@
 user_00025: 
   salt: 2aa64c10d3ea67188845a59d3d9d411d52301838
   created_at: 2009-12-12 21:07:52 Z
-  activated_at: 2009-12-12 21:08:07 Z
+  confirmed_at: 2009-12-12 21:08:07 Z
   recently_reset: false
   crypted_password: acfecdcd9a91ee01aa7da07d29d3ffb6036714fa
```

Comment:
I believe this (and all similar lines in this file) needs to be changed to `encrypted_password` instead of `crypted_password`.